python/cudnn/sdpa/fwd/api_dsl.py

```
@@ -1619,6 +1642,102 @@ def _scalar(t, default=1.0):
+        meta.copy_(torch.tensor(slk_host + cu_q_host + cu_k_host, dtype=torch.int32))
+        t_q = cu_q_host[-1]
+        t_kv = cu_k_host[-1]
+        if t_q == 0:
+            self._logger.debug("execute (FP8 THD): t_q == 0, nothing to do")
+            return
+
+        qh, kh = self.h_q, self.h_kv
+        d_qk, d_v = self.head_dim_qk, self.head_dim_v
+
+        Q = self._thd_view(q_buf, self.q_desc, t_q)
+        O = self._thd_view(o_buf, self.o_desc, t_q)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)